src/lv_misc/lv_mem.c

```diff
@@ -85,7 +85,7 @@ typedef struct {
 #endif
 
 static uint32_t zero_mem; /*Give the address of this variable if 0 byte should be allocated*/
-
+static uint32_t mem_max_size; /*Tracks the maximum total size of memory allocated from the internal heap*/ 
```

Comment:
```suggestion
static uint32_t mem_max_size; /*Tracks the maximum total size of memory ever used from the internal heap*/ 
```

Reply:
It seems clearer to me. What do you think?

Reply:
Seems Ok, i was struggling with the correct naming in English.